chapter11/compose.override.yaml

```diff
@@ -49,5 +49,11 @@ services:
     # yamllint enable rule:line-length
     user: "0:0"
 
+  movie-reviews:
+    build: ./docker/movie-reviews-api
+    image: manning-airflow/events-api
```

Comment:
Name is different from build context directory. I would align them. I believe events-api might be already use din other chapters so I suggest to use the reviews-api as name

Reply:
good catch, fixed.